app/controllers/concerns/find_common.rb

```diff
@@ -22,12 +22,18 @@ def resource_scope(options = {})
     @resource_scope ||= begin
       options[:controller] ||= controller_name
       options[:permission] ||= "#{action_permission}_#{options[:controller]}"
-      scope = resource_class.scoped
-      if resource_class.respond_to?(:authorized)
+      scope = resource_class.all
+      scope = if resource_class.respond_to?(:authorized)
         scope.authorized(options[:permission], resource_class)
       else
         scope
       end
+
+      if scope.respond_to?(:friendly)
```

Comment:
testing it would mean testing the scope mechanism, which is silly, as it's fairly well tested. scopes, by their definition, don't override each other.